### ISSUE TITLE
SDK should have its own project

### ISSUE THREAD

**Issue by itsrachelfish** [wetfish/login#3]
The SDK for wetfish login is designed to be used in other applications, however because it is included in this project it is technically licensed under the GPL. For broader accessibility, the SDK should be MIT licensed in a separate repo, and should be made available on npm.

**Comment by itsrachelfish** [comment]
The SDK has been moved to its own repo and has been relicensed as MIT.

https://github.com/wetfish/login-sdk
